bokeh/settings.py

```diff
@@ -218,6 +232,14 @@ def nodejs_path(self, default=None):
 
 #: A global settings object that other parts of Bokeh can refer to.
 #:
+#:
+#: ``BOKEH_BINARY_ARRAY_CUTOFF`` --- Number of samples before arrays are encoded.
+#:
+#:   Valid values include all positive integers. Ensures that binary array
+#:   encoding is only applied to large arrays, avoiding the overhead
+#:   in small cases. Only applies when ``BOKEH_USE_BINARY_ARRAYS`` is
+#:   enabled.
```

Comment:
Why not hardcode this to something reasonable, having trouble imagining it being important to tune this?